package.json

```diff
@@ -9,14 +9,12 @@
     "build:css": "node-sass css/styles.scss lib/css/_datepicker.css",
     "build:svg": "webpack",
     "clean": "rimraf lib",
-    "cover": "npm run cover:clean && npm run cover:mocha && npm run cover:check",
-    "cover:clean": "rimraf coverage",
-    "cover:check": "babel-node ./node_modules/istanbul/lib/cli check-coverage",
-    "cover:mocha": "babel-node ./node_modules/istanbul/lib/cli cover node_modules/mocha/bin/_mocha -- --compilers js:babel-register,jsx:babel-register --require airbnb-js-shims --recursive ./test/_helpers test/**/*",
+    "precover": "rimraf coverage",
+    "cover": "cross-env NODE_ENV=test nyc npm run mocha",
     "lint": "eslint --ext .js,.jsx src test",
-    "mocha": "mocha --compilers js:babel-register,jsx:babel-register --require airbnb-js-shims --recursive ./test/_helpers",
+    "mocha": "mocha ./test/_helpers test/**/*_spec.js*",
```

Comment:
is the `_spec.js` part needed? Every JS file inside `test` should be considered a test, and double-running the helpers shouldn't be a problem :-/